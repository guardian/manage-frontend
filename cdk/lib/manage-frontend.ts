import { join } from 'path';
import type { GuStackProps } from '@guardian/cdk/lib/constructs/core';
import { GuStack } from '@guardian/cdk/lib/constructs/core';
import type { App } from 'aws-cdk-lib';
import { CfnInclude } from 'aws-cdk-lib/cloudformation-include';

export class ManageFrontend extends GuStack {
	constructor(scope: App, id: string, props: GuStackProps) {
		super(scope, id, props);
	}
}
