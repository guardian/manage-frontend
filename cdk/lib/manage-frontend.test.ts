import { App } from 'aws-cdk-lib';
import { Template } from 'aws-cdk-lib/assertions';
import { ManageFrontend } from './manage-frontend';

describe('The ManageFrontend stack', () => {
	it('matches the snapshot', () => {
		const app = new App();
		const stack = new ManageFrontend(app, 'ManageFrontend', {
			stack: 'support',
			stage: 'TEST',
		});
		const template = Template.fromStack(stack);
		expect(template.toJSON()).toMatchSnapshot();
	});
});
