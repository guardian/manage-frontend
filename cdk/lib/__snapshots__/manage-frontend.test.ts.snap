// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The ManageFrontend stack matches the snapshot 1`] = `Object {}`;
